extensions-core/histogram/src/main/java/org/apache/druid/query/aggregation/histogram/FixedBucketsHistogram.java

```diff
@@ -431,6 +433,33 @@ public void incrementMissing()
     }
   }
 
+  /**
+   * Merge another datapoint into this one. The other datapoin could be
+   *  - base64 encoded string of {@code FixedBucketsHistogram}
+   *  - {@code FixedBucketsHistogram} object
+   *  - Numeric value
+   *
+   * @param val
+   */
+  void combine(@Nullable Object val)
```

Comment:
Good point. I will let this method remain here. Since I am only tackling numeric values for now, my vector implementation can call `add` directly on fixed histogram. will make that change.